google/resource_compute_instance_group_manager.go

```diff
@@ -132,18 +132,18 @@ func resourceComputeInstanceGroupManagerCreate(d *schema.ResourceData, meta inte
 		return err
 	}
 
-	// Get group size, default to 1 if not given
-	var target_size int64 = 1
+	var targetSize int64 = 0
 	if v, ok := d.GetOk("target_size"); ok {
-		target_size = int64(v.(int))
+		targetSize = int64(v.(int))
 	}
 
 	// Build the parameter
 	manager := &compute.InstanceGroupManager{
 		Name:             d.Get("name").(string),
 		BaseInstanceName: d.Get("base_instance_name").(string),
 		InstanceTemplate: d.Get("instance_template").(string),
-		TargetSize:       target_size,
+		TargetSize:       targetSize,
+		ForceSendFields:  []string{"TargetSize"},
```

Comment:
might be nice to have a comment here like "Force send TargetSize to allow it to have a value of 0"